cmake/ClangTools.cmake

```diff
@@ -21,27 +21,22 @@ endif(CMAKE_CROSSCOMPILING)
 # Check for Clang format
 set(CLANG_FORMAT_PATH "NOTSET" CACHE STRING "Absolute path to the clang-format executable")
 if("${CLANG_FORMAT_PATH}" STREQUAL "NOTSET")
-    find_program(CLANG_FORMAT NAMES
-        clang-format39 clang-format-3.9
-        clang-format38 clang-format-3.8
-        clang-format37 clang-format-3.7
-        clang-format36 clang-format-3.6
-        clang-format35 clang-format-3.5
-        clang-format34 clang-format-3.4
-        clang-format)
-    message(STATUS "Using clang format: ${CLANG_FORMAT}")
-    if("${CLANG_FORMAT}" STREQUAL "CLANG_FORMAT-NOTFOUND")
-        message(WARNING "Could not find 'clang-format' please set CLANG_FORMAT_PATH:STRING")
-    else()
-        set(CLANG_FORMAT_PATH ${CLANG_FORMAT})
-        message(STATUS "Found: ${CLANG_FORMAT_PATH}")
-    endif()
+  find_program(CLANG_FORMAT NAMES
+    clang-format-3.8
```

Comment:
?  I thought we went 4.0 and 6.0?  I can't even get 3.8 on 18.04 without sacrificing chickens...

Reply:
Yeah, it was originally 3.8 and below, then at the beginning of this PR it was switched and I just reverted back to 3.8 since I can't get 4.0 to install on travis.  https://github.com/swift-nav/albatross/issues/100